docs/sources/flow/reference/components/prometheus.exporter.mssql.md

```diff
@@ -0,0 +1,106 @@
+---
+# NOTE(rfratto): the title below has zero-width spaces injected into it to
+# prevent it from overflowing the sidebar on the rendered site. Be careful when
+# modifying this section to retain the spaces.
+#
+# Ideally, in the future, we can fix the overflow issue with css rather than
+# injecting special characters.
+
+title: prometheus.exporter.mssql
+---
+
+# prometheus.exporter.mssql
+The `prometheus.exporter.mssql` component embeds
+[sql_exporter](https://github.com/burningalchemist/sql_exporter) for collecting stats from a Microsoft SQL Server.
+
+## Usage
+
+```river
+prometheus.exporter.mssql "LABEL" {
+  connection_string = "CONNECTION_STRING"
+}
+```
+
+
+## Arguments
+
+The following arguments can be used to configure the exporter's behavior.
+Omitted fields take their default values.
+
+| Name                   | Type       | Description                                                       | Default | Required |
+|------------------------|------------|-------------------------------------------------------------------|---------|----------|
+| `connection_string`    | `secret`   | The connection string used to connect to an Microsoft SQL Server. |         | yes      |
+| `max_idle_connections` | `int`      | Maximum number of idle connections to any one target.             | `3`     | no       |
+| `max_open_connections` | `int`      | Maximum number of open connections to any one target.             | `3`     | no       |
+| `timeout`              | `duration` | The query timeout in seconds.                                     | `"10s"` | no       |
+
+
+
+``The `connection_string` argument`` should be in the [format](https://github.com/burningalchemist/sql_exporter/blob/master/examples/azure-sql-mi/sql_exporter.yml#L21):
```

Comment:
It's best to indicate in the text where the link is taking someone, so something like: "[The sql_exporter examples](https://github.com/burningalchemist/sql_exporter/blob/master/examples/azure-sql-mi/sql_exporter.yml#L21) shows the format of the `connection_string` argument: [...]" (Not meant to be a direct suggestion, just an example :sweat_smile:).

And are these double backticks intended? 
```suggestion
The `connection_string` argument should be in the [format](https://github.com/burningalchemist/sql_exporter/blob/master/examples/azure-sql-mi/sql_exporter.yml#L21):
```

Reply:
Noted, updated to suggested example. Thanks!